src/app/icd/server/ICDManager.h

```diff
@@ -131,6 +139,12 @@ class ICDManager : public ICDListener
     void OnSubscriptionReport() override;
 
 protected:
+    /**
+     * @brief Hepler function that extends the Active Mode duration by the extendDuration parameter
+     *        as well as the Active Mode Jitter timer for the transition to iddle mode.
```

Comment:
This comment doesn't reflect what the function done.

This says :
currentTimer = 15s,
extend by 30s
new timer is 45s

when it does
current Timer 15s
extend to 30s
new timer 30s

Reply:
Agreed but that behavior is dictated by the SystemLayer Implementation, so I think we cannot indicate the precise behavior here as it is the SystemLayer that decides in the end. I'll just avoid being specific about the exact extension.